terraform/vars.tf

```diff
@@ -42,6 +48,18 @@ variable "kms_location" {
   default = "us-central1"
 }
 
+# Name of the key ring for export signing keys.
+variable "kms_export_signing_key_ring_name" {
```

Comment:
Do we also want to update their usage in this PR? https://github.com/google/exposure-notifications-server/blob/c1975364369fe6d8b02485656bd3b08c9549d62f/terraform/key_management.tf#L17

Reply:
Done